### ISSUE TITLE
Enhancement: Auto submit form after filling username and password - one click login

### ISSUE THREAD

**Issue by SylwesterZarebski** [bitwarden/browser#129]
I would like to see enhancement which allows to automatically submit form after filling username and password. It could work similar to Magic Wand from old Opera, which allows to one click login.

**Comment by dolfje** [comment]
Yes, definitely. Only not if two-factor authentication is enabled.

**Comment by SylwesterZarebski** [comment]
Correct link is:
https://community.bitwarden.com/t/automatically-submit-login-form-on-auto-fill/24

**Comment by kspearrin** [comment]
Whoops .
